### DISCUSSION TITLE
Prometheus metrics - Trace , span derived metrics

### DISCUSSION THREAD

**Discussion by devrimdemiroz** [open-telemetry/opentelemetry-java-instrumentation#2168] [Q&A]
Hi,

I have seen the thread https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/1880 as answered 'the java agent is not producing metrics yet'

I have seen jvm metrics in another discussion. However, I wonder if is there any plan to export potential metrics of instrumentation. To be more precise, similar measurements like Jaeger provides per trace under "Trace Statistics" e.g. "Operation Name" count, duration.

Thanks,
Devrim

**Comment by trask** [comment]
hey @devrimdemiroz! yes, there is a whole OpenTelemetry metrics specification and metrics semantic conventions that are being worked on over in the opentelemetry-specification repo, and we will be implementing at some point in the javaagent
